cmd/runtimetest/main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/mount"
```

Comment:
I'm not familiar with Docker's internal package organization, but I worry about cyclic dependencies when a low level tool like ocitools pulls in code from a high-level spec consumer like docker/docker.  Can generic mount parsing/handling get pulled out into an independent repository somewhere so that it's clear that it's also low-level (if it is, I haven't looked)?